src/NBitcoin/Consensus.cs

```diff
@@ -23,7 +23,7 @@ public class Consensus : IConsensus
         public Money ProofOfStakeReward { get; }
 
         /// <inheritdoc />
-        public uint MaxReorgLength { get; }
+        public uint MaxReorgLength { get; private set; }
```

Comment:
`private set;` is redundant and inconsistent with other fields in the class.

Reply:
We need this to set values on Consensus via reflection.

Reply:
for some test? why value of 500 isn't good?

Reply:
Mining 500 blocks in integration tests takes too long duded ; I dont see an issue with a private setter. We need in anycase look into being able to set consensus settings in test after the network has been created.